src/util/Options.js

```diff
@@ -149,6 +153,53 @@ class Options extends null {
     };
   }
 
+  /**
+   * Create a cache factory using predefined sweep settings.
+   * @param {Record<string, SweptCollectionOptions>} [settings={}] Settings for structures.
+   * @returns {CacheFactory}
+   */
+  static cacheWithSweep(settings = {}) {
+    const Collection = require('./Collection');
+    const SweptCollection = require('./SweptCollection');
+
+    return manager => {
+      const setting = settings[manager.name];
+      if (
+        setting === null || setting === undefined || typeof setting !== 'object' || setting.sweepInterval
```

Comment:
`typeof setting !== 'object'` should already exclude undefined.
```suggestion
        setting === null || typeof setting !== 'object' || setting.sweepInterval
```